README.md

```diff
@@ -1,12 +1,60 @@
-# The Lounge
+<h1 align="center">
+	<img
+		width="300"
+		alt="The Lounge"
+		src="client/img/logo-vertical-transparent-bg.svg">
```

Comment:
This could link to image from website repository.

Reply:
I hesitated between link to the main website or nothing, and went for nothing because it's just too random :) Plus we already have a link to the website underneath. But link to the website repo, I'm not a fan, it's plain weird

Reply:
I meant image link

Reply:
Ah, right, but it's already a link to the image (within this repo) so :man_shrugging:

Reply:
We wouldn't have to track the logo separately in this repo, just like we don't do it for the screenshot.

Reply:
Oh wow, I realize I totally misread what you meant from the beginning. I get it now.

I'm going to use that same image file in the app itself, so I'd have to add it anyway.